igor-web/src/main/groovy/com/netflix/spinnaker/igor/admin/AdminController.java

```diff
@@ -15,63 +15,67 @@
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-import java.util.stream.Collectors;
-
-import static java.lang.String.format;
-
 @RestController
 @RequestMapping("/admin")
 public class AdminController {
 
-    private final Logger log = LoggerFactory.getLogger(getClass());
+  private final Logger log = LoggerFactory.getLogger(getClass());
 
-    private final List<CommonPollingMonitor> pollingMonitors;
+  private final List<CommonPollingMonitor> pollingMonitors;
 
-    @Autowired
-    public AdminController(Optional<List<CommonPollingMonitor>> pollingMonitors) {
-        this.pollingMonitors = pollingMonitors.orElseGet(ArrayList::new);
-    }
+  @Autowired
+  public AdminController(Optional<List<CommonPollingMonitor>> pollingMonitors) {
+    this.pollingMonitors = pollingMonitors.orElseGet(ArrayList::new);
+  }
 
-    /**
-     * Silently fast-forwards a poller. Fast-forwarding means that all pending cache state will be polled and saved,
-     * but will not send Echo notifications.
-     *
-     * By default all partitions (ex: masters) will be fast-forwarded, however specific partition names should be
-     * used whenever possible.
-     *
-     * @param monitorName The polling monitor name (ex: "DockerMonitor")
-     * @param partition The partition name, if not provided, method will re-index the entire monitor
-     */
-    @RequestMapping(value = "/pollers/fastforward/{monitorName}", method = RequestMethod.POST)
-    @ResponseStatus(HttpStatus.NO_CONTENT)
-    public void fastForward(@PathVariable String monitorName,
-                        @RequestParam(required = false) String partition) {
-        CommonPollingMonitor pollingMonitor = pollingMonitors.stream()
+  /**
+   * Silently fast-forwards a poller. Fast-forwarding means that all pending cache state will be
+   * polled and saved, but will not send Echo notifications.
+   *
+   * <p>By default all partitions (ex: masters) will be fast-forwarded, however specific partition
+   * names should be used whenever possible.
+   *
+   * @param monitorName The polling monitor name (ex: "DockerMonitor")
+   * @param partition The partition name, if not provided, method will re-index the entire monitor
+   */
+  @RequestMapping(value = "/pollers/fastforward/{monitorName}", method = RequestMethod.POST)
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  public void fastForward(
+      @PathVariable String monitorName, @RequestParam(required = false) String partition) {
+    CommonPollingMonitor pollingMonitor =
+        pollingMonitors.stream()
             .filter(it -> it.getName().equals(monitorName))
             .findFirst()
-            .orElseThrow(() -> new NotFoundException(format(
-                "PollingMonitor %s was not found, available monitors are: %s",
-                monitorName,
-                pollingMonitors.stream().map(PollingMonitor::getName).collect(Collectors.toList())
-            )));
+            .orElseThrow(
+                () ->
```

Comment:
this section is not as nice as it was before, I will admit.

Reply:
Too bad!